src/main/java/org/retest/rebazer/connector/BitbucketConnector.java

```diff
@@ -124,10 +121,12 @@ private DocumentContext jsonPathForPath( final String urlPath ) {
 
 	@Override
 	public void addComment( final PullRequest pullRequest, final String message ) {
-		final Map<String, String> request = new HashMap<>();
-		request.put( "content", message );
+		final ObjectNode contentNode = objectMapper.createObjectNode();
+		final ObjectNode contentNode1 = objectMapper.createObjectNode();
+		contentNode1.put( "raw", message );
+		contentNode.set( "content", contentNode1 );
```

Comment:
I would suggest to change the variables names and statements order. For instance:

```java
final ObjectNode comment = objectMapper.createObjectNode();
comment.put( "raw", message );
final ObjectNode request = objectMapper.createObjectNode();
request.set( "content", comment );
```

If you can derive better names from the actual API, that's also good.

Reply:
sry about that the names were somehow not adopted...